src/mmw/js/src/modeling/models.js

```diff
@@ -38,7 +39,9 @@ var ProjectModel = Backbone.Model.extend({
         this.set('model_package', 'tr-55');
         this.set('taskModel', new Tr55TaskModel());
 
-        this.makeFirstScenarioActive();
+        // After the scenario collection is initialized, set
+        // the first scenario as the active scenario.
+        this.listenToOnce(this, 'change:scenarios', this.makeFirstScenarioActive);
```

Comment:
I was planning on treating modifications as immutable so we probably don' need to listen for property changes. We could just set `area_acres` once upon initialization.